provider/resources.go

```diff
@@ -57,7 +58,7 @@ var metadata []byte
 func Provider() tfbridge.ProviderInfo {
 	// Instantiate the Terraform provider
 	p := pfbridge.MuxShimWithPF(context.Background(),
-		pfbridge.ShimProvider(buildkite.New(version.Version)),
+		shimv2.NewProvider(buildkite.Provider()),
```

Comment:
Ref: https://github.com/buildkite/terraform-provider-buildkite/pull/497

@muhlba91 I would like to ask if you are okay with my approach of adding that `buildkite.Provider()` helper

Reply:
sure, just go ahead. 😃